course/templates/course/reviews_by_course.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'base_home.html' %}
```

Comment:
Maybe extend the loggedInPage so the side navbar will appear

Reply:
Done